src/assets/stylesheets/style.css

```diff
@@ -0,0 +1,276 @@
+/*** Project Meta ***/
+
+/* general - line 13 */
+
+/* header - line 39 */
+
+/* main content - line 19 */
+
+/* footer - line 144 */
+
+/** GENERAL **/
+
+/* @font-face {
+  font-family: 'MyFont';
+  src: url('./my-font.woff2') format('woff2'),
+    url('./my-font.woff') format('woff');
+  font-weight: 600;
+  font-style: normal;
+} */
+
+/* :root {
+  --border-bottom: 1px solid rgb(230, 230, 230);
+  --grey: rgb(246, 246, 246);
+  --flex: flex;
+  --border-radius: 10px;
+} */
+
+/* Mobile View */
+
+.html {
```

Comment:
- [ ] OPTIONAL: I highly suggest you align these "comment" buttons.  Your app will look a lot better:

![image](https://user-images.githubusercontent.com/57421823/164952610-2bc6535f-64f9-4ada-bceb-a206aca50b49.png)